CHANGELOG.md

```diff
@@ -1,7 +1,7 @@
 # 2.10.1 (2020-11-06)
 
 ### Bugfix
-- By default any item should be selected [niglesias]
+- By default no item should be selected [niglesias]
```

Comment:
Sorry I didn't see this on first review, but you should be explicit about what you fixed:
```
- **Sidenav demo**:
  - By default no item should be selected [niglesias]
  - Clicking on pastanaga icon in the sidenav should display home page again [niglesias]
  - on mobile/tablet, selecting a sidenav item should close the menu [niglesias]
```

Reply:
oh true, i wasn't sure to use sidenav since the fixes are not for the sidenav component, thank you!